Add restore command (#244)
thijsvanloef/palworld-server-docker: README.md

The server will run a save before the backup if rcon is enabled.

## Restore a backup

To restore a backup, use the command:

```bash
docker exec palworld-server restore
```

> The container must be restarted after the recovery progresses.
> 
> > [!IMPORTANT]
> > The `RCON_ENABLED` environment variable must be `true` to use this command.

Reviewer: Modified readme to be more inline with [Configuring Automatic Updates with Cron](https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#configuring-automatic-updates-with-cron)
```suggestion
The `RCON_ENABLED` environment variable must be set to `true` to use this command.
> [!IMPORTANT]
> If docker restart is not set to policy `always` or `unless-stopped` then the server will shutdown and will need to be
> manually restarted.
>
> The example docker run command and docker compose file in [How to Use](#how-to-use) already uses the needed policy
```

The server will run a save before the backup if rcon is enabled.

## Restore a backup

To restore a backup, use the command:

```bash

> docker exec palworld-server restore

Reviewer: ```suggestion
docker exec -i palworld-server restore
```
This needs to be interactive

> docker exec -i palworld-server restore

Reviewer: ```suggestion
docker exec -it palworld-server restore
```

I really thought only `-i` was needed but that resulted in this
```
Backup List:
1) /palworld/backups/blah.tar.gz
2) /palworld/backups/palworld-save-2024-02-02_00-21-18.tar.gz
3) /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
#? 3
Selected backup: /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
Do you want to continue with the command?
y
```

Changing it to `-it` gave the expected

```
Backup List:
1) /palworld/backups/blah.tar.gz
2) /palworld/backups/palworld-save-2024-02-02_00-21-18.tar.gz
3) /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
#? 3
Selected backup: /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
Do you want to continue with the command?
When you run it, the server will be stopped and the recovery will proceed. (y/n): y
```

Author: 조금더 잘확인했어야했는데 제실수입니다.
빠른 정정 감사합니다.
